packages/date/src/index.js

```diff
@@ -560,6 +560,21 @@ export function getDate( dateString ) {
 	return momentLib.tz( dateString, WP_ZONE ).toDate();
 }
 
+/**
+ * Returns the difference between two timestamps in a human-readable format.
+ *
+ * @param {Moment | Date | string | undefined} from         The timestamp to measure from. Defaults to the current time if undefined or invalid.
+ * @param {Moment | Date | string | undefined} to           The timestamp to measure to. Defaults to the current time if undefined or invalid.
+ * @param {boolean}                            includeAffix Whether to include the "ago" or "to" affix in the comparison.
+ * @return {string} The difference between the timestamps.
+ */
+export function humanTimeDiff( from, to, includeAffix ) {
+	// Normalize inputs to a moment object, defaults to current time.
+	from = momentLib( from );
+	to = momentLib( to );
+	return from.from( to, ! includeAffix );
```

Comment:
I think it's worth it, in order to avoid mutations and help readability, to define `to` and `from` as new constants with a distinct name.